sbi/neural_nets/factory.py

```diff
@@ -275,8 +305,28 @@ def build_fn(batch_theta, batch_x):
             return build_maf_rqs(batch_x=batch_theta, batch_y=batch_x, **kwargs)
         elif model == "nsf":
             return build_nsf(batch_x=batch_theta, batch_y=batch_x, **kwargs)
+        elif model == "mnle":
+            return build_mnle(batch_x=batch_theta, batch_y=batch_x, **kwargs)
+        elif model == "zuko_nice":
+            return build_zuko_nice(batch_x=batch_theta, batch_y=batch_x, **kwargs)
         elif model == "zuko_maf":
             return build_zuko_maf(batch_x=batch_theta, batch_y=batch_x, **kwargs)
+        elif model == "zuko_nsf":
```

Comment:
These are not covered by tests.

Reply:
can be easily added to the ones in `tests/neural_nets_factory.py`